tsdb/index/tsi1/file_set.go

```diff
@@ -38,13 +38,7 @@ func (fs *FileSet) bytes() int {
 
 // Close closes all the files in the file set.
 func (fs FileSet) Close() error {
-	var err error
```

Comment:
Moving this is just a refactor right?

Reply:
I hope so. I actually would like, in a perfect world, to get rid of FileSet and just use Files as the type, but that's not a worthwhile refactor at the moment